espaloma/mm/nonbonded.py

```diff
@@ -119,21 +138,20 @@ def lj_9_6(x, sigma, epsilon):
     )
 
 
-#
-# def columb(x, q_prod, k_e=K_E):
-#     """ Columb interaction without cutoff.
-#
-#     Parameters
-#     ----------
-#     x : `torch.Tensor`, shape=`(batch_size, 1)` or `(batch_size, batch_size, 1)`
-#     q_prod : `torch.Tensor`,
-#         `shape=(batch_size, 1) or `(batch_size, batch_size, 1)`
-#
-#     Returns
-#     -------
-#     u : `torch.Tensor`,
-#         `shape=(batch_size, 1)` or `(batch_size, batch_size, 1)`
-#
-#
-#     """
-#     return k_e * x / q_prod
+def coulomb(x, q, k_e=K_E):
+    """ Columb interaction without cutoff.
+
+    Parameters
+    ----------
+    x : `torch.Tensor`, shape=`(batch_size, 1)` or `(batch_size, batch_size, 1)`
+    q_prod : `torch.Tensor`,
+        `shape=(batch_size, 1) or `(batch_size, batch_size, 1)`
+
+    Returns
+    -------
+    u : `torch.Tensor`,
+        `shape=(batch_size, 1)` or `(batch_size, batch_size, 1)`
+
+
+    """
+    return 0.5 * k_e * q / x
```

Comment:
You might document that this is supposed to be half the Coulomb interaction since both i-j and j-i interactions will be summed.